### PR TITLE
Completed style, create, edit trello task

### PR DESCRIPTION
**Pull request by don-yoon**

### CODE REVIEW

File: app/javascript/components/pages/LastCallEdit.js [hunk 1]
```diff
@@ -1,169 +1,189 @@
...
 
-  let [cocktail, setCocktail] = useState([])
-  
+  let [cocktail, setCocktail] = useState([]);
 
-  console.log("show:", logged_in)
-  console.log("data", cocktail.user_id)
-  function showCocktails (id) {
+  console.log("show:", logged_in);
+  console.log("data", cocktail.user_id);
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/LastCallEdit.js]: Do you need these console.logs?

File: app/javascript/components/pages/LastCallEdit.js [hunk 2]
```diff
@@ -1,169 +1,189 @@
...
-    editCocktail(cocktailEdit, id)
-    setSubmit(true)
+    e.preventDefault();
+    editCocktail(cocktailEdit, id);
+    setSubmit(true);
   }
 
   function handleCocktailArray(ing, e) {
-    e.preventDefault()
+    e.preventDefault();
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/LastCallEdit.js]: e.preventDefault is only needed for forms.
for regular buttons you don't need it.

event handler (e) isin't being used anywhere in this function; remove it.

File: app/javascript/components/pages/LastCallEdit.js [hunk 3]
```diff
@@ -1,169 +1,189 @@
...
                       <Form.Control
-                        type="text" placeholder="Ingredients"
+                        type="text"
+                        placeholder="Ingredients"
                         onChange={(e) => setIngredient(e.target.value)}
                       />
                     </Form.Group>
-                    <Button onClick={(e) => handleCocktailArray(ingredient, e)}>Add Ingredient</Button>
+                    <Button onClick={(e) => handleCocktailArray(ingredient, e)}>
+                      Add Ingredient
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/LastCallEdit.js:104]: don't need (e) as a param

File: app/javascript/components/pages/LastCallNew.js [hunk 4]
```diff
@@ -1,154 +1,166 @@
-import React from 'react'
-import { useState, useRef } from 'react'
-import { Form, Button, Container, Col, Row } from 'react-bootstrap'
-import { Navigate } from 'react-router-dom'
+import React from "react";
+import { useState, useRef } from "react";
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/LastCallNew.js]: combine these imports

File: app/javascript/components/pages/LastCallNew.js [hunk 5]
```diff
@@ -1,154 +1,166 @@
...
+import { Form, Button, Container, Col, Row } from "react-bootstrap";
+import { Navigate } from "react-router-dom";
 
 export default function LastCallNew(props) {
+  let { createCocktail, current_user, logged_in } = props;
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/LastCallNew.js:6]: restructuring can also be done on the line above from props.

Comment (don-yoon) [reply app/javascript/components/pages/LastCallNew.js:6]: How would i get current_user, it's being passed as props from rails devise?

Comment (iameddieyayaya) [reply app/javascript/components/pages/LastCallNew.js:6]: ment destructuring 

```export default function LastCallNew({createCocktail, current_user, logged_in}) {```

^ is the same thing as line 6

**Review by iameddieyayaya — APPROVED** [review]
LGTM 🚀

**Pull request merged**